src/bundle/Resources/views/admin/content_type/edit.html.twig

```diff
@@ -113,8 +126,8 @@
         </div>
     </section>
 
-    {{ form_widget(form.publishContentType, { 'attr': { 'hidden': 'hidden' }}) }}
```

Comment:
Please do not remove quotes from the keys, this is the standard we are following in Twig templates.